json_serializable/test/integration/integration_test.dart

```diff
@@ -304,6 +304,14 @@ void main() {
     expect(dayTypeEnumValues, ['no-good', 'rotten', 'very-bad']);
   });
 
+  test('enum serialization has correct type', () {
+    // ignore: omit_local_variable_types
+    final Issue1145Regression cls =
+        Issue1145Regression(status: {Issue1145RegressionEnum.gamma: true});
+    // Not Map<String?, dynamic>
```

Comment:
Drop this comment. JSON does not allow null keys (at least I don't think so)

Reply:
I'm happy to clarify the comment, but checking that the resultant output is _not_ Map<String**?**, dynamic> is  the intent of the test